server/src/main/java/io/unitycatalog/server/service/ModelService.java

```diff
@@ -137,6 +137,10 @@ public HttpResponse createModelVersion(@AuthorizeKeys({
           @AuthorizeKey(value = REGISTERED_MODEL, key = "model_name")})
```

Comment:
@creechy With source being moved to required, I don't need to alter these annotations at all right?  This are purely for ACL checks on the actual securable keys?

Reply:
Yeah you shouldn't need to alter the annotations at all unless you want to include source in the ACL checking somehow.

Reply:
Nah.  I just wanted to double check on this.